docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -0,0 +1,110 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Redis Reference Guide
+
+:extension-status: preview
+include::./attributes.adoc[]
+:numbered:
+:sectnums:
+
+[[custom_redis_commands]]
+== How to use custom Redis Commands
+
+As the list of commands which are supported out-of-box by both https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisClient.java[`RedisClient`] and https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/reactive/ReactiveRedisClient.java[`ReactiveRedisClient`] depends on what is available in https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`], then there might be case when you need a command which is not (yet) available via https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`].
+
+[NOTE]
+====
+Please see these issues for the initial idea:
+
+https://github.com/quarkusio/quarkus/issues/18351
+
+https://github.com/vert-x3/vertx-redis-client/issues/299
+====
+
+In such case (if you don't want to wait for the new command to be supported in https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`]), you can implement it in either https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisClient.java[`RedisClient`] or https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/reactive/ReactiveRedisClient.java[`ReactiveRedisClient`]. 
+In order to do so, you will need:
+
+- Generate a new `Command` based on the nodejs code available in https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`] repository:
```

Comment:
Having a list with only one item is weird. Maybe just `need to ...`.

Reply:
@gsmet Actually it's not a single item list, but five items:

- Generate a new `Command` based on the nodejs code ...
- Updating the RedisClient ...
- Updating the RedisClientImpl ...
- Updating the ReactiveRedisClient ...
- Updating the ReactiveRedisClientImpl ...

The problem is that the 1st bullet is quite extensive, so it might confuse readers.